terraform/cloudfoundry/vpc_peering.tf

```diff
@@ -0,0 +1,27 @@
+resource "aws_vpc_peering_connection" "vpc_peer" {
+  count         = "${length(var.peer_vpc_ids)}"
+  peer_owner_id = "${element(var.peer_account_ids, count.index)}"
+  peer_vpc_id   = "${element(var.peer_vpc_ids, count.index)}"
+  vpc_id        = "${var.vpc_id}"
+}
+
+resource "aws_route" "vpc_peer_route_0" {
+  count                     = "${length(var.peer_vpc_ids)}"
+  route_table_id            = "${aws_route_table.internet.0.id}"
+  destination_cidr_block    = "${element(var.peer_cidrs, count.index)}"
+  vpc_peering_connection_id = "${element(aws_vpc_peering_connection.vpc_peer.*.id, count.index)}"
+}
```

Comment:
These are currently flapping (being added and removed on alternate terraform runs). This looks to be because we're combining `aws_route` resources with inline route definitions ([here](https://github.com/alphagov/paas-cf/blob/master/terraform/cloudfoundry/nat.tf#L16-L19)). This is documented to cause problems - https://www.terraform.io/docs/providers/aws/r/route.html